### PR TITLE
Fix output directory of test

### PR DESCRIPTION
**Pull request by disktnk**
after

```
out/opset7
├── test_absolute
│   ├── model.onnx
│   └── test_data_set_0
│       ├── input_0.pb
│       └── output_0.pb
├── test_add
│   ├── model.onnx
│   └── test_data_set_0
│       ├── input_0.pb
│       ├── input_1.pb
│       └── output_0.pb
├── test_addconstant
│   ├── model.onnx
│   └── test_data_set_0
│       ├── input_0.pb
│       └── output_0.pb
├── test_average_pooling_2d
...
```

Apply only function test, I'll do it for `test_rt.py` and `test_inout.py` on next PR.

### CODE REVIEW

File: onnx_chainer/export.py [hunk 1]
```diff
@@ -166,7 +166,8 @@ def backward_postprocess(self, function, in_data, out_grad):
 
 
 def export(model, args, filename=None, export_params=True,
-           graph_name='Graph', save_text=False, opset_version=None):
+           graph_name='Graph', save_text=False, opset_version=None,
+           return_flat_inout=False):
```
Comment (shinh) [inline onnx_chainer/export.py]: Could you update the docstring?

File: onnx_chainer/testing/test_onnxruntime.py [hunk 2]
```diff
@@ -23,6 +23,53 @@
...
 
 
+def check(model, args, name, opset_version):
+    test_path = gen_test_data_set(model, args, name, opset_version)
+    check_model_expect(test_path)
+
+
+def gen_test_data_set(model, args, name, opset_version):
+    model.xp.random.seed(42)
+    test_path = os.path.join(
```
Comment (shinh) [inline onnx_chainer/testing/test_onnxruntime.py]: Just a note: I don't think we should do something immediately, but I guess now we output a few giga bytes to "out" directory? Perhaps it's better to auto-remove some ONNX models generated by test_rt.py. For example, I'd keep large ONNX models only for the most recent opset_version.

File: onnx_chainer/export.py (hunk 1 above)
Comment (disktnk) [reply onnx_chainer/export.py]: fixed.

File: onnx_chainer/testing/test_onnxruntime.py (hunk 2 above)
Comment (disktnk) [reply onnx_chainer/testing/test_onnxruntime.py]: agree, pytest framework support command args, for example, add `pytest --keep-models=False` option easily (maybe), I'll survey these options. Thanks.

File: onnx_chainer/export.py [hunk 3]
```diff
@@ -196,6 +197,8 @@ def export(model, args, filename=None, export_params=True,
             or ``None`` is given, the latest opset version of the onnx module
             is used. If an integer is given, it will be ensured that all the
             operator version in the exported ONNX file is less than this value.
+        return_flat_inout (bool): If set True, return ONNX model with flat
+            inputs, and flat outputs.
```
Comment (shinh) [inline onnx_chainer/export.py]: Could you also update `Returns`? Also, it'd be better to clarify the types of flattened inputs/outputs in the Returns section

Comment (disktnk) [reply onnx_chainer/export.py]: add detail return doc, thx.

**Pull request merged**